crates/core/tedge_api/src/entity_store.rs

```diff
@@ -314,12 +347,22 @@ impl EntityStore {
                 }
             }
         };
+
+        let JsonValue::Object(mut other) = message.other else {
+            return Err(Error::EntityRegistrationOtherNotMap);
+        };
+
+        // XXX: this is c8y-specific, entity store shouldn't do this!
```

Comment:
What difference between `XXX` and `FIXME`?

Reply:
I think they can be understood to be roughly the same, I personally use `XXX` to highlight code which i think is problematic and should be addressed relatively quickly [after seeing some other people use it this way](https://drewdevault.com/2023/03/09/2023-03-09-Comment-or-no-comment.html). I've also seen both `XXX` and `FIXME` used in thin-edge, so I used `XXX`. If you consider `FIXME` to be more descriptive, then I can use that instead.

Here, as `serviceType` is something only used by c8y, handling it this way works for now, but it will be unsustainable if we keep adding properties, so this should probably be addressed some other way and refactored in the future.

Reply:
I never noticed usage of XXX and I'm used to FIXME. I'm okay with both: the key point is to agree as a team.